challenges/largest-product/largest_array.py

```diff
@@ -0,0 +1,17 @@
+def LargestArray(arr):
+    
+"""
+Finds the largest product in a 2D array
+"""
+    answer = 0
+    counter = 0
+    new_arr = []
+
+    for item in arr:
+        new_arr[counter] = item[0] * item[1]
+        counter += 1
+    for item in new_arr:
+        if item > answer:
+            answer = item
+    return answer
```

Comment:
This only checks for products of number next to each other horizontally. You also need to check for values next to each other vertically.
```
[[1, 2], [3, 4], [5, 6], [7, 8]]
```
is equivalent to the matrix
```
1  2
3  4
5  6
7  8
```
So you need to multiply `1 * 2` as well as `1 * 3` in order to find the max.